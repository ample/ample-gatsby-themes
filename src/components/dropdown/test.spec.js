import React from "react"
import renderer from "react-test-renderer"

import Dropdown from "./"

describe("Dropdown", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Dropdown trigger={{ label: "Trigger" }}>
          <ul></ul>
        </Dropdown>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
