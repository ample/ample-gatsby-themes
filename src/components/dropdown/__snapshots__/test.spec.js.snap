// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown renders correctly 1`] = `
<span
  className="dropdown"
  onBlur={[Function]}
>
  <a
    className="dropdown_trigger"
    href=""
    onClick={[Function]}
    rel="noopener"
    target="_blank"
  >
    <span>
      Trigger
    </span>
    <span>
      <svg
        height={8}
        viewBox="0 0 11 8"
        width={11}
      >
        <path
          d="M9.524.724l1.262 1.263L5.393 7.38 0 1.987 1.263.724l4.13 4.131z"
          fillRule="evenodd"
        />
      </svg>
    </span>
  </a>
  <div
    className="dropdown_menu"
    onMouseDown={[Function]}
  >
    <ul />
  </div>
</span>
`;
